### PR TITLE
change configuration to the new Kin playground and production environments

### PR DESCRIPTION
**Pull request by yosriz**
…ments

### CODE REVIEW

File: sample/src/main/java/kin/core/sample/OnBoarding.java [hunk 1]
```diff
@@ -19,10 +19,10 @@
 
 class OnBoarding {
 
-    private static final String URL_CREATE_ACCOUNT = "http://friendbot-kik.kininfrastructure.com/?addr=";
+    private static final String URL_CREATE_ACCOUNT = "http://friendbot-playground.kininfrastructure.com/?addr=";
     private static final int FUND_KIN_AMOUNT = 6000;
     private static final String URL_FUND =
-        "http://159.65.84.173:5000/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+        "http://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
```
Comment (shayb) [inline sample/src/main/java/kin/core/sample/OnBoarding.java:25]: @yosriz why do the client needs it?
are you sure this needs to be exposed

Comment (yosriz) [reply sample/src/main/java/kin/core/sample/OnBoarding.java:25]: for sample app, sample app has an onboarding example that uses friendbot/facuet to initiate account

Comment (shayb) [reply sample/src/main/java/kin/core/sample/OnBoarding.java:25]: if this is only the for the sample app and this should not be exposed as part of the sdk maybe better write it hardcoded in the sample app?

Comment (yosriz) [reply sample/src/main/java/kin/core/sample/OnBoarding.java:25]: it is hardcoded in the sample app, that's part of the sample app

**Review by berryve — APPROVED** [review]

**Pull request merged**